common/src/main/scala/explore/components/TileController.scala

```diff
@@ -105,7 +103,7 @@ object TileController {
       }
       // Store the current layout
       .useStateViewWithReuseBy((p, _, _) => updateResizableState(p.layoutMap))
-      .renderWithReuse { (p, debouncer, bn, currentLayout) =>
+      .render { (p, debouncer, bn, currentLayout) =>
```

Comment:
In this case for example, it seems the Reuse does not complicate the code a lot

Reply:
I started by removing it from everywhere, and have a working version of Explore with that.

We can add it back in the places where it's needed, but I think we should have a uniform criterion.

For example, I notice the performance is not the same as before when resizing. We should probably add it back in the situations where it does have an impact.